packages/react/src/combobox/ComboboxBase/ComboboxBase.tsx

```diff
@@ -112,6 +115,15 @@ export function ComboboxBase<Option extends DefaultComboboxOption>({
 
 	const { id: labelId } = combobox.getLabelProps();
 
+	const handleOnBlur = (event: FocusEvent<HTMLInputElement>) => {
+		// If a user presses the Done button on the iOS keyboard, we shouldn't close the keyboard
```

Comment:
```suggestion
		// If a user presses the Done button on the iOS dropdown, we shouldn't close the keyboard
```

Reply:
Looks like you need to change the other `keyboard` instance in your sentence:

```suggestion
		// If a user presses the Done button on the iOS keyboard, we shouldn't close the dropdown
```